src/App/App.css

```diff
@@ -0,0 +1,67 @@
+.App {
```

Comment:
I would go ahead and delete these files since it is not providing any styles to the application